exporters/trace/src/test/java/com/google/cloud/opentelemetry/trace/TraceTranslatorTest.java

```diff
@@ -0,0 +1,198 @@
+package com.google.cloud.opentelemetry.trace;
+
+import com.google.devtools.cloudtrace.v2.AttributeValue;
+import com.google.devtools.cloudtrace.v2.Span;
+import com.google.devtools.cloudtrace.v2.TruncatableString;
+import com.google.rpc.Status;
+import io.opentelemetry.common.Attributes;
+import io.opentelemetry.common.ReadableAttributes;
+import io.opentelemetry.sdk.trace.data.SpanData;
+import io.opentelemetry.trace.Span.Kind;
+import io.opentelemetry.trace.*;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.*;
+import java.util.concurrent.TimeUnit;
+
+import static org.junit.Assert.*;
+
+@RunWith(JUnit4.class)
+public class TraceTranslatorTest {
+
+  @Test
+  public void toDisplayNameTest(){
+    String serverPrefixSpanName = "Recv. mySpanName";
+    String clientPrefixSpanName = "Sent. mySpanName";
+    String regularSpanName = "regularSpanName";
+    Kind serverSpanKind = Kind.SERVER;
+    Kind clientSpanKind = Kind.CLIENT;
+
+    assertEquals("Recv. mySpanName", TraceTranslator.toDisplayName(serverPrefixSpanName, serverSpanKind));
+    assertEquals("Sent. mySpanName", TraceTranslator.toDisplayName(clientPrefixSpanName, clientSpanKind));
+    assertEquals("Recv.regularSpanName", TraceTranslator.toDisplayName(regularSpanName, serverSpanKind));
+    assertEquals("Sent.regularSpanName", TraceTranslator.toDisplayName(regularSpanName, clientSpanKind));
+  }
+
+  @Test(expected = NullPointerException.class)
+  public void nullTruncatableStringProtoTest(){
+    TruncatableString nullTruncatable = TraceTranslator.toTruncatableStringProto(null);
+
+    assertNull(nullTruncatable.getValue());
+    assertEquals(0, nullTruncatable.getTruncatedByteCount());
+  }
+
+  @Test
+  public void toTruncatableStringProtoTest(){
+    String truncatableString = "myTruncatableString";
+    TruncatableString testTruncatable = TraceTranslator.toTruncatableStringProto(truncatableString);
+
+    assertEquals("myTruncatableString", testTruncatable.getValue());
+    assertEquals(0, testTruncatable.getTruncatedByteCount());
```

Comment:
when would it not be 0?

Reply:
It would be nonzero if there were bytes somehow removed from the original string

Reply:
Is there a way to test that case?

Reply:
probably not with regards to `toTruncatableStringProto` actually, because the method specifies setting `truncatedByteCount` to 0.

`return TruncatableString.newBuilder().setValue(string).setTruncatedByteCount(0).build();`

Reply:
Looks good to me then :)